adaway-linux.sh

```diff
@@ -51,8 +52,19 @@ mkdir -p "${TMPDIR}"
 
 # add domains from hosts-server listet in hostssources.lst
 while read src; do
+    timeout=20
+    retries=2
```

Comment:
why did you not move this to the top and made it also `readonly`?

Reply:
I didn't considered the variables to be a global setting, this has been changed now to your preference. ;)